advanced/keyboard-shortcuts.md

```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
+| Save                 | Ctrl+S                | Command+S                   | [save]({{ site.baseurl }}/plugins/save/)                   |
+| Find                 | Ctrl+F                | Command+F                   | [searchreplace]({{ site.baseurl }}/plugins/searchreplace/) |
 
 ## Accessibility keyboard shortcuts
 
 This is a list of available keyboard shortcuts within the editor user interface.
 
-| Action | Key |
-|--------|-----|
-| Execute command | Enter / Space |
-| Focus next widget | Right Arrow / Tab / Down Arrow |
-| Focus previous widget | Left Arrow / Shift+Tab / Up Arrow |
-| Open widget menu | Down Arrow |
-| Open submenu | Right Arrow |
-| Close submenu | Left Arrow |
-| Close dialog | Esc |
-| Close menu | Esc |
-| Move focus back to editor body | Esc |
+| Action                                                                           | PC                       | Mac                      |
+|----------------------------------------------------------------------------------|--------------------------|--------------------------|
+| Execute command                                                                  | Enter / Space            | Enter / Space            |
+| Focus to menu bar                                                                | Alt+F9                   | Option+F9                |
+| Focus to toolbar                                                                 | Alt+F10                  | Option+F10               |
+| Focus to element path                                                            | Alt+F11                  | Option+F11               |
+| Focus to contextual toolbar                                                      | Ctrl+F9                  | Ctrl+F9                  |
+| Focus on next UI Element<br>(such as: Menu bar, Toolbar, Toolbar Group, Status Bar Item)     | Tab                      | Tab                      |
+| Focus on previous UI Element<br>(such as: Menu bar, Toolbar, Toolbar Group, Status Bar Item) | Shift+Tab                | Shift+Tab                |
+| Focus next Control<br>(such as: toolbar button, menu, or menu item)                  | Right Arrow / Down Arrow | Right Arrow / Down Arrow |
+| Focus previous Control<br>(such as: toolbar button, menu, or menu item)              | Left Arrow / Up Arrow    | Left Arrow / Up Arrow    |
+| Open menu, toolbar menu button, split toolbar button, or group toolbar button    | Down Arrow               | Down Arrow               |
+| Open the popup menu on split toolbar buttons                                     | Shift+Enter              | Shift+Enter              |
+| Open submenu                                                                     | Right Arrow              | Right Arrow              |
+| Close submenu                                                                    | Left Arrow               | Left Arrow               |
```

Comment:
Left Arrow || Escape

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1543/commits/b294e937ae83eb64451f079631b6b16d17bfaac8